python/vineyard/deploy/utils.py

```diff
@@ -116,3 +119,26 @@ def start_etcd(host=None, etcd_executable=None):
         print('Etcd being killed...')
         if proc is not None and proc.poll() is None:
             proc.terminate()
+
+
+def delete_namespace(namespace):
+    kubernetes.config.load_kube_config()
+    k8s_core_v1 = kubernetes.client.CoreV1Api()
+    resp = k8s_core_v1.delete_namespace(namespace)
+    print("Delete Namespace. status=%s" % resp)
+
+
+def start_etcd_k8s(namespace):
+    kubernetes.config.load_kube_config()
+    with open(os.path.join(os.path.dirname(__file__), 'etcd.yaml')) as f:
+        dep = yaml.safe_load(f)
+        k8s_core_v1 = kubernetes.client.CoreV1Api()
+        resp = k8s_core_v1.create_namespaced_pod(body=dep, namespace=namespace)
+        print("Etcd Pods created. status=%s" % resp.metadata.name)
+
+    with open(os.path.join(os.path.dirname(__file__), 'etcd_service.yaml')) as f:
+        dep = yaml.safe_load(f)
+        print(dep)
+        k8s_core_v1 = kubernetes.client.CoreV1Api()
+        resp = k8s_core_v1.create_namespaced_service(body=dep, namespace=namespace)
+        print("Etcd Service created. status=%s" % resp.metadata.name)
```

Comment:
Why not create pod and service in the same yaml?

See: https://github.com/kubernetes-client/python/blob/master/kubernetes/utils/create_from_yaml.py